packages/liveblocks-client/src/client.ts

```diff
@@ -37,11 +37,11 @@ export function createClient(options: ClientOptions): Client {
     return internalRoom ? internalRoom.room : null;
   }
 
-  function enter<TStorageRoot>(
+  function enter<TStorage>(
```

Comment:
This is a consistency pass, so all the type params are named the same everywhere.